tests/WidgetTest.php

```diff
@@ -117,13 +150,25 @@ public function testInjection(): void
         $this->assertInstanceOf(Injectable::class, $widget->getInjectable());
     }
 
-    public function testFactoryExceptionWidget(): void
+    public function testWidgetThrownExceptionForNotInitializeWidgetFactory(): void
```

Comment:
May be we can way run this test first (configure PhpUnit) and don't user reflection ?

Reply:
I don't see the point in redoing all the others because of one case.